python/packaging/__main__.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+import sys
+from pathlib import Path
+
+F3D_BIN_PATH = Path(__file__).parent / "bin" / "f3d"
+if sys.platform == "win32":
+    F3D_BIN_PATH = F3D_BIN_PATH.with_suffix(".exe")
+
+def run(args):
+    return subprocess.call([F3D_BIN_PATH] + args)
+
+def main():
+    return SystemExit(run(sys.argv[1:]))
```

Comment:
```suggestion
def main():
    exit(subprocess.call([F3D_BIN_PATH] + sys.argv[1:]))
```

Reply:
What's the difference?

Reply:
not much really :) However now I'm questioning a lot more about it...

`__main__.py` runs the F3D app a process from a python script, either when running `python -m f3d` or just `f3d` that would have been setup in the local/virtualenv `bin/` directory by `setup.py` (`entry_points={"console_scripts": ["f3d=f3d.__main__:main"]}`).
That works because `setup.py` actually builds the `pybind` module _and the whole f3d app_ and installs it in `site-packages`.

But is that really the goal, or should the python module only contain the libf3d python bindings and not the F3D app?

Reply:
Yes, currently it builds everything. We could remove the executable but the initial PR description #319 suggest to keep it.  
We could even rely on a pre-installed libf3d on build only the bindings, it's supported

Reply:
ah yes I've skimmed over the original PR too fast and missed this key bit:

> I advocate for including the f3d binary in the python distribution wheel, to me pip the fastest & easiest way to install software (especially if I only need it for a specific project).

The goal is to use pip as a general package manager / distribution method to deliver both the whole F3D app and the libf3d bindings. I'm not totally sold on the soundness of the idea but it is definitely valid, moving on :)